machine_configuration/100-intel-qat-intel-iommu-on.yaml

```diff
@@ -12,6 +12,6 @@ spec:
     ignition:
       version: 3.2.0
   kernelArguments:
-      - intel_iommu=on
+      - intel_iommu=on vfio-pci.ids=8086:4941,8086:4943
```

Comment:
is this enough or `modules_load=vfio-pci` needed too?

Reply:
Yes those were needed. updated the change and verified in our cluster